### PR TITLE
Refactor StepWizardAgent

### PR DESCRIPTION
**Pull request by sarthak-marwaha**
- Refactor StepWizardAgent
- Add ElementStepWizardAgent and FlowAgentWizardlet
- Update existing StepWizardAgents

### CODE REVIEW

File: src/foam/u2/wizard/debug/TestWizard.js [hunk 1]
```diff
@@ -30,6 +30,15 @@ foam.CLASS({
...
 
+        scenarioMenus.push(this.Menu.create({
+          id: this.id + '/ElementWizardTest',
+          label: 'Element Wizard Test',
+          parent: this.id,
+          handler: foam.nanos.menu.ViewMenu.create({
+            view: { class: 'net.nanopay.cards.test.wizards.ElementTest' }
+          }, this)
+        }));
+
```
Comment (KernelDeimos) [inline src/foam/u2/wizard/debug/TestWizard.js:41]: I'm acknowledging that this is here and it's not easy to move somewhere else. I added NP-7141 to track this separately.

File: src/foam/u2/wizard/views/FlexibleWizardContentsView.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ foam.CLASS({
     function render() {
       const self = this;
       const current$ = this.slot(function (data, data$currentWizardlet, data$currentSection) {
-        return data$currentSection.createView();
+        return data$currentSection?.createView() ?? this.E();
```
Comment (KernelDeimos) [inline src/foam/u2/wizard/views/FlexibleWizardContentsView.js:57]: that's a lotta ES6 in one place :D

File: src/foam/u2/wizard/wizardlet/FlowAgentWizardlet.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/**
+* @license
+* Copyright 2022 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.u2.wizard.wizardlet',
+  name: 'FowAgentWizardlet',
```
Comment (KernelDeimos) [inline src/foam/u2/wizard/wizardlet/FlowAgentWizardlet.js]: Fow or Flow?

File: src/foam/u2/wizard/views/FlexibleWizardContentsView.js (hunk 2 above)
Comment (sarthak-marwaha) [reply src/foam/u2/wizard/views/FlexibleWizardContentsView.js:57]: Didn't have other _options_ in this case

**Pull request merged**